src/wrapped/wrappedvulkan_private.h

```diff
@@ -312,6 +312,7 @@ GO(vkGetPhysicalDeviceMemoryProperties2KHR, vFpp)
 GO(vkGetPhysicalDeviceProperties2KHR, vFpp)
 GO(vkGetPhysicalDeviceQueueFamilyProperties2KHR, vFppp)
 GO(vkGetPhysicalDeviceSparseImageFormatProperties2KHR, vFpppp)  //VkSparseImageFormatProperties2 seems OK
+GO(vkGetPhysicalDeviceExternalTensorPropertiesARM, vFpp)
```

Comment:
not the correct signature (vFppp) and bad placement, it's the VK_ARM_tensors extension